game.py

```diff
@@ -0,0 +1,41 @@
+from exceptions import *
```

Comment:
The better practice is not to use * in import. You can use import exceptions and use exceptions.GameOver for example